packages/ilp-spsp-payout/src/index.ts

```diff
@@ -26,6 +28,7 @@ export class Payout {
       this.createPlugin = makeIlpPlugin
     }
     this.payouts = {}
+    this.slippage = opts && opts.slippage
```

Comment:
The build is failing here with `##[error]ilp-spsp-payout: src/index.ts(31,5): error TS2322: Type 'number | undefined' is not assignable to type 'number'.`.